src/main/views/et-daily-list.njk

```diff
@@ -0,0 +1,146 @@
+      <p>{{ openJustice5 }}</p>
+      <p>{{ openJustice6 }}</p>
+    </div>
+  </details>
+  {% for courtList in listData['courtLists'] %}
+    {% set courtHouse = courtList['courtHouse'] %}
+    <div class="site-header" tabindex="0" aria-label="Court Details">
+      <h1 class="govuk-heading-l site-address" tabindex="0">{{ venue }}{{ courtHouse['courtHouseName'] }}</h1>
+      {% set courtHouseAddress = courtHouse['courtHouseAddress'] %}
+      {% for line in courtHouseAddress['line'] %}
+        <p class="govuk-body site-address" tabindex="0">{{ line }}</p>
+      {% endfor %}
+      {% if courtHouseAddress['town'] | length %}
+        <p class="govuk-body site-address" tabindex="0">{{ courtHouseAddress['town'] }}</p>
+      {% endif %}
+      {% if courtHouseAddress['county'] | length %}
+        <p class="govuk-body site-address" tabindex="0">{{ courtHouseAddress['county'] }}</p>
+      {% endif %}
+      {% if courtHouseAddress['postCode'] | length %}
+        <p class="govuk-body site-address" tabindex="0">{{ courtHouseAddress['postCode'] }}</p>
+      {% endif %}
+    </div>
+
+    <div class="govuk-accordion" data-module="govuk-accordion" id="accordion-default">
+      {% set courtRoomCount = 0 %}
+      {% for courtRoom in courtHouse['courtRoom'] %}
+        {% set hearingCount = 1 %}
+        {% set courtRoomCount = courtRoomCount + 1 %}
+        <div class="govuk-accordion__section govuk-accordion__section--expanded">
+          <div class="govuk-accordion__section-header">
+            <h2 class="govuk-accordion__section-heading">
+            <span class="govuk-accordion__section-button" id="accordion-default-heading-{{ courtRoomCount }}">
+              {{ courtRoom['courtRoomName'] }}
+            </span>
+            </h2>
+
+          </div>
+          <div id="accordion-default-content-{{ courtRoomCount }}" class="govuk-accordion__section-content"
+               aria-labelledby="accordion-default-heading-{{ courtRoomCount }}">
+            <div class="parent-box overflow-table">
+              <table class="govuk-table">
+                <thead class="govuk-table__head">
+                <tr class="govuk-table__row">
+                  {% for header in tableHeaders %}
+                    <th scope="col" class="govuk-table__header">{{ header }}</th>
+                  {% endfor %}
+                </tr>
+                </thead>
+                <tbody class="govuk-table__body">
+                {% for session in courtRoom['session'] %}
+                  {% for sitting in session['sittings'] %}
+                    {% set durationAsHours = 0 %}
+                    {% if sitting['durationAsHours'] > 1 %}
+                      {% set durationAsHours = sitting['durationAsHours'] + ' ' + hearingDurationHours %}
+                      {% elif sitting['durationAsHours'] === 1 %}
```

Comment:
This elif is not indented correctly

Reply:
Interesting stuff! Looks like a quirk of the intelliJ plugin for njk files.